### PR TITLE
Add ecs-logging-go-zap to the build

### PR DESCRIPTION
**Pull request by bmorelli25**
Adds `ecs-logging-go-zap` to the build.

For https://github.com/elastic/ecs-logging-go-zap/issues/20.

### CODE REVIEW

File: shared/attributes.asciidoc [hunk 1]
```diff
@@ -89,6 +89,7 @@ endif::[]
 :sql-odbc:             https://www.elastic.co/guide/en/elasticsearch/sql-odbc/{branch}
 :ecs-ref:              https://www.elastic.co/guide/en/ecs/{ecs_version}
 :ecs-logging-ref:      https://www.elastic.co/guide/en/ecs-logging/overview/{ecs-logging}
+:ecs-logging-go-zap-ref:  https://www.elastic.co/guide/en/ecs-logging/java/{ecs-logging-go-zap}
```
Comment (bmorelli25) [inline shared/attributes.asciidoc:92]: fix copy pasta
```suggestion
:ecs-logging-go-zap-ref:  https://www.elastic.co/guide/en/ecs-logging/go-zap/{ecs-logging-go-zap}
```

**Review by simitt — APPROVED** [review]

**Review by gtback — APPROVED** [review]
LGTM!

**Comment by gtback** [comment]
@bmorelli25 is the plan to merge this in as soon as elastic/ecs-logging-go-zap#29 is merged?

**Comment by bmorelli25** [comment]
Yup!

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Pull request merged**